defaults/main.yml

```diff
@@ -46,6 +46,10 @@ devture_traefik_config_log_level: INFO
 # Controls whether logs for incoming requests are collected
 devture_traefik_config_accessLog_enabled: true
 
+# Controls whether prometheus metrics will be exposed. You will need to set
+# a entrypoint named `metrics` for this to work properly
+devture_traefik_config_metrics_prometheus_enabled: false
```

Comment:
This makes it sound like you will need to manually need to configure additional entrypoints.

From what I understand, the `metrics` entrypoint will be created automatically, wouldn't it?

Perhaps the following comment will be more suitable:

```suggestion
# Controls whether Prometheus metrics will be exposed on a new metrics entrypoint.
# See devture_traefik_config_entrypoint_metrics_enabled
devture_traefik_config_metrics_prometheus_enabled: false
devture_traefik_config_metrics_prometheus_entrypoint: metrics
```

Note the addition of a new `devture_traefik_config_metrics_prometheus_entrypoint` variable, for controlling the `metrics.prometheus.entryPoint` configuration setting.

Reply:
Ahh yes, the comment was left over from a solution on which I decided against